### PR TITLE
OAuthProxy update - #1409

### PR DESCRIPTION
**Pull request by KurtStam**
https://github.com/syndesisio/syndesis/issues/1409

### CODE REVIEW

File: content/quickstart.md [hunk 1]
```diff
@@ -60,6 +60,14 @@ Depending on your role please use the appropriate template in the instructions b
...
 ### Deployment instructions
 
+Before we can start installing Syndesis we have to add some policies
+
+```bash
+$ oc login -u system:admin
+$ oc adm policy add-cluster-role-to-user system:auth-delegator -z syndesis-oauth-client
+$ oc login -u developer
+```
+
```
Comment (rhuss) [inline content/quickstart.md]: @KurtStam @zregvart I thought we reverted this requirement ?

Comment (zregvart) [reply content/quickstart.md]: Yeah we did by removing the functionality 😄

Comment (paoloantinori) [reply content/quickstart.md]: I think the 2 operations (requirement removal and doc update) happened more or less at the same time.

Comment (rhuss) [reply content/quickstart.md]: @KurtStam so can we please remove this section again ?

**Pull request merged**